AutoPrognosis.md

```diff
@@ -0,0 +1,35 @@
+# What is AutoPrognosis?
+AutoPrognosis is an algorithmic framework for automating the design of machine learning based clinical prognostic models[1]. Helps in choosing which algorithms to use and tune parameters.
+
+# Source code of AutoPrognosis
+https://github.com/ahmedmalaa/AutoPrognosis 
+
+# Data set used
+UK Biobank dataset (http://www.ukbiobank.ac.uk/register-apply/). 
+Dataset contains data from a prospective cohort study group of 423,604 participants without CVD.
+
+# Components of AutoPrognosis. 
+![schematic image](https://github.com/jrlvarghese/Ayevh/blob/master/autoprognosis_schematic.PNG)
```

Comment:
Kindly refer source of this image. In case it is your drawing, then kindly mention as such in reply.